### ISSUE TITLE
Does not run on android

### ISSUE THREAD

**Issue by pogofdev** [pradeep1991singh/ImageColorPickerExampleApp#1]
Hi!
Is android supported?. The script seems to be broken when running on Webview.

**Comment by rjuevesano** [comment]
Following....

**Comment by pradeep1991singh** [comment]
Expect a fix by the weekend.

**Comment by rjuevesano** [comment]
hi @pradeep1991singh do you have fixes already?

**Comment by pradeep1991singh** [comment]
@rjuevesano Yes, I have released the fix, check react-native-image-color-picker@1.0.4 and update example app too.

**Comment by rjuevesano** [comment]
![image](https://user-images.githubusercontent.com/4033035/40036390-c94ffa48-5839-11e8-9e7e-80fd97ce7c9a.png)

i got that error ^
did i missed something?

**Comment by pradeep1991singh** [comment]
@rjuevesano did you linked the new dependency, make sure you are using `react-native-image-color-picker@1.0.4` and after that run `react-native link image-color-picker`.

**Comment by rjuevesano** [comment]
Yes. I did that already but still gives me that error above.

**Comment by pradeep1991singh** [comment]
@rjuevesano send me the device details and meantime try example app and see if that works.

**Comment by rjuevesano** [comment]
samsung s8 android 7.0.0 and i try to get the colors for this image link https://driverapp.asiacarservice.com/img/4880848542236672?key=3f0acd64248c4d25babe31bbd41597ae&ts=1526365498599
